README.md

```diff
@@ -1,19 +1,45 @@
-# Cyrcle
-The Bike Companion
+# Cyrcle - The Bike Companion
 
-## Description
-Cyrcle is a community-driven app designed to solve the everyday challenges faced by  cyclists. <br>
-Our mapping system helps you find the perfect parking spot and repair stations for your bike,
-ensuring that your two-wheeled companion is always safe and well-maintained.
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/46704d8172b7434da89388f53748cf43)](https://app.codacy.com/gh/SwEnt-Fall-2024-Group-22/Cyrcle/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
+[![Codacy Badge](https://app.codacy.com/project/badge/Coverage/46704d8172b7434da89388f53748cf43)](https://app.codacy.com/gh/SwEnt-Fall-2024-Group-22/Cyrcle/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_coverage)
+![GitHub license](https://img.shields.io/github/license/SwEnt-Fall-2024-Group-22/Cyrcle)
+![GitHub forks](https://img.shields.io/github/forks/SwEnt-Fall-2024-Group-22/Cyrcle)
+![GitHub stars](https://img.shields.io/github/stars/SwEnt-Fall-2024-Group-22/Cyrcle)
+
+Cyrcle is a community-driven app designed to solve the everyday challenges faced by cyclists. <br>
+Our mapping system helps you find the perfect parking spot for your bike, ensuring that your 
+two-wheeled companion is always safe well-stored.
```

Comment:
You probably forgot an `and` here at the end of the sentence? As in:
"Our mapping system helps you find the perfect parking spot for your bike, ensuring that your 
two-wheeled companion is always safe **and** well-stored."

Reply:
Thank you for the catch. It is fixed !